src/spacesettlers/utilities/Movement.java

```diff
@@ -12,18 +12,19 @@
 	 * Acceleration in orientation
 	 */
 	double angularAccleration;
-	
+
 	/**
 	 * Accelerating in x and y (x'' and y'' if you prefer)
 	 */
 	Vector2D translationalAcceleration;
-	
+
 	/**
-	 * Maximum accelerations.  If you specific one above (or below the negative), it will set to the max
+	 * Maximum accelerations. If you specific one above (or below the negative), it
+	 * will set to the max
 	 */
-	static public final double MAX_TRANSLATIONAL_ACCELERATION = 50;
+	static public final double MAX_TRANSLATIONAL_ACCELERATION = 62;
```

Comment:
Adjusted to account for the acceleration previously falling between `50` and `50 * Math.sqrt(2)` so that ships don't struggle even more with turning etc.